02_bubble/jessodri/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	b := append([]int{}, s...)
+	length := len(b)
+	for i := 0; i < length; i++ {
+		for j := 0; j < (length - i - 1); j++ {
```

Comment:
Since `i` is only used to calculate the limit for this inner loop and nowhere else, you could turn the outer loop around to make this `(length - i -1)` a little easier to comprehend.

That is:
```
for i := length-1; i >= 0; i-- {
    for j := 0; j < i; j++ {
        ...
    }
}
```

From this I think it is easier to see that `i` counts down from the end of the loop and `j` counts up to `i` a little more clearly than figuring out that `(length - i - 1)` really does the same thing.

Just a suggestion, and no need to change this.